wifiphisher/pywifiphisher.py

```diff
@@ -595,6 +595,10 @@ def start(self):
                        "rogue Access Point").format(G, W, ap_iface.get_name())
 
             kill_interfering_procs()
+            #randomize the adapters MAC addresses
+            self.network_manager.randomize_interface_mac(ap_iface)
+            self.network_manager.randomize_interface_mac(mon_iface)
+
             self.network_manager.set_interface_mode(mon_iface, "monitor")
```

Comment:
This is out of your patch, but i'm wondering why it's here. If user runs the script with the `--nojamming` it will get the one and only interface to `monitor` while we need it to `AP`. 

CC: @blackHatMonkey